auslib/web/balrog_api.py

```diff
@@ -0,0 +1,76 @@
+import connexion
+from connexion.operation import Operation
+from connexion.resolver import Resolver
+
+
+RANDOM_DIGITS = 6
+
+
+class BalrogApi(connexion.Api):
```

Comment:
The feature to reuse function name (OperationId) is disabled by default. So I had to inherit from base connexion classes.